xbmc/FileItem.cpp

```diff
@@ -4032,5 +4032,30 @@ bool CFileItem::GetCurrentResumeTimeAndPartNumber(int64_t& startOffset, int& par
 
 bool CFileItem::IsResumable() const
 {
-  return (!IsNFO() && !IsPlayList()) || IsType(".strm");
+  if (m_bIsFolder)
```

Comment:
Did you check this new implementation against strm files?

Reply:
Yes, I did.